### ISSUE TITLE
Add clickable default prepend-URL for logo:0:url

### ISSUE THREAD

**Issue by phavekes** [OpenConext/OpenConext-manage#437]
[This issue is imported from pivotal - Originaly created at May 16, 2024 by andreklaver](https://www.pivotaltracker.com/story/show/187618146)

Logo URL\'s are basically one of two:
https://static.surfconext.nl/media/idp/logo.png
https://static.surfconext.nl/media/sp/logo.png

Currently we copy and paste the static part of the URL from god knows somewhere when a new SP/IdP is created. 

Request: add a button in Manage on the line or input field of logo:0:url
 so a user of Manage can click to prepend the SURFconext Static Part or replace the not SURFconext Static part as defined by an SP or RP to \'https://static.surfconext.nl/media/sp/\' for SP\'s/RP\'s or \'https://static.surfconext.nl/media/idp/\' for IdP\'s

**Comment by phavekes** [comment]
I would pause this for a bit since we\'re currently planning to have SP\'s upload logos themselves via SP Dashboard. For IdPs it does not change too often. (Thijs Kinkhorst - Jun 4, 2024)

**Comment by baszoetekouw** [comment]
Agreed.  We need to make the entire process easier, rather than patching this small detail.  This will most definitely be solved in the new ABA dashboard, and also the manual backend process will need to change because the logo uploading will need to be moved anyway.  Closing this for now.
